src/App.tsx

```diff
@@ -1,21 +1,25 @@
 import React from 'react';
 import { ThemeProvider } from '@mui/material/styles';
 import { Main } from './Main';
-import { theme } from './theme';
+import { theme, adminTheme } from './theme';
 import { RecoilRoot } from 'recoil';
-import { Route, Routes } from "react-router-dom";
+import { Route, Routes, useLocation } from "react-router-dom";
 import LoginPage from './login';
 import AdminPage from './admin';
 import { RequireAuth } from './RequireAuth';
 import { AuthProvider } from './AuthProvider';
-
+import CssBaseline from '@mui/material/CssBaseline';
 
 const App: React.FC = () => {
 
+  const adminThemeMap = ["/login", "/admin"];
```

Comment:
This variable name is a bit confusing, should this be a map (Or maybe a Set)? Would make the check below instant time lookup